specification/cognitiveservices/data-plane/Personalizer/preview/v2022-09-01-preview/Personalizer.json

```diff
@@ -1,2640 +1,2642 @@
-      "type": "object",
-      "properties": {
-        "ranking": {
-          "description": "The calculated ranking for the current request.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/RankedAction"
-          },
-          "readOnly": true
-        },
-        "eventId": {
-          "description": "The eventId for the round trip from request to response.",
-          "maxLength": 256,
-          "type": "string",
-          "readOnly": true
-        },
-        "rewardActionId": {
-          "description": "The action chosen by the Personalizer service.\r\nThis is the action your application should display, and for which to report the reward.\r\nThis might not be the first found in \u0027ranking\u0027.",
-          "maxLength": 256,
-          "type": "string",
-          "readOnly": true
-        }
-      }
-    },
-    "ServiceStatus": {
-      "type": "object",
-      "properties": {
-        "service": {
-          "type": "string"
-        },
-        "apiStatus": {
-          "type": "string"
-        },
-        "apiStatusMessage": {
-          "type": "string"
-        }
-      }
-    }
-  },
-  "parameters": {
-    "Endpoint": {
-      "in": "path",
-      "name": "Endpoint",
-      "description": "Supported Cognitive Services endpoint.",
-      "required": true,
-      "type": "string",
-      "x-ms-parameter-location": "client",
-      "x-ms-skip-url-encoding": true
-    }
-  },
-  "securityDefinitions": {
-    "apim_key": {
-      "type": "apiKey",
-      "name": "Ocp-Apim-Subscription-Key",
-      "in": "header"
-    }
-  },
-  "security": [
-    {
-      "apim_key": []
-    }
-  ],
-  "tags": [
-    {
-      "name": "ConfigurationsV1Dot1Preview1",
-      "description": "Manages configuration operations."
-    },
-    {
-      "name": "MultiSlotEvents",
-      "description": "Manages multi-slot event operations."
-    },
-    {
-      "name": "MultiSlotRank",
-      "description": "Manages multi-slot ranking operations."
-    },
-    {
-      "name": "Evaluations20220901Preview",
-      "description": "Manages evaluations operations."
-    },
-    {
-      "name": "FeatureImportances20220901Preview",
-      "description": "Manages feature importance operations."
-    },
-    {
-      "name": "Events",
-      "description": "Manages event operations."
-    },
-    {
-      "name": "Rank",
-      "description": "Manages ranking operations."
-    },
-    {
-      "name": "LogsV1Dot1Preview2",
-      "description": "Manages reinforcement learning logs."
-    },
-    {
-      "name": "ModelV1Dot1Preview3",
-      "description": "Manages reinforcement learning configuration operations."
-    }
-  ],
-  "x-ms-parameterized-host": {
-    "hostTemplate": "{Endpoint}",
-    "useSchemePrefix": false,
-    "parameters": [
-      {
-        "$ref": "#/parameters/Endpoint"
-      }
-    ]
-  }
-}
+	"swagger": "2.0",
```

Comment:
This diff is not reviewable. It all appears "changed" because the indentation has changed. Please ensure you have your spacing set to 2 spaces to match the previous formatting.

Reply:
I fixed the formatting in the latest commits. Personalizer.json is a new file added to v2022-09-01-preview folder in this PR. 
Not sure what I should do about the older commit. @tjprescott